src/uint.rs

```diff
@@ -215,17 +201,6 @@ impl UInt {
         self.value.is_zero()
     }
 
-    /// Returns `true` if this `UInt` represents the value one (`1`).
-    ///
-    /// # Note
-    ///
-    /// - One (`1`) is also called the multiplicative neutral element.
-    /// - This operation is more efficient than comparing two instances of
-    ///   `UInt` for the same reason.
-    pub fn is_one(&self) -> bool {
-        self.value.is_one()
-    }
-
```

Comment:
Same here since `Uint::one` yields a proper value for all bit widths we should support `is_one` as well.